frontend/src/entities/group/detail/group-detail.tsx

```diff
@@ -104,17 +108,34 @@ export function GroupDetail () {
                     content: (
                         <Table
                             tableName='Dataset'
-                            actions={groupDatasetUserActions}
+                            actions={GroupDetailDatasetActions}
                             itemNameProperty='name'
                             trackBy='location'
-                            allItems={groupDatasets}
+                            allItems={groupDatasets || []}
```

Comment:
Why is this change needed? We aren't doing it for projects or users

Reply:
We don't need that anymore. I think it was leftover from the many revisions I went through on this.